bin/cmake-js

```diff
@@ -102,7 +102,7 @@ var yargs = require("yargs")
             describe: "the architecture to build in",
             type: "string"
         },
-        o: {
+        p: {
```

Comment:
We must not break existing behavior in favor of a new one. Please use another letter for the new option.